examples/events/basicevent/basicevent.go

```diff
@@ -0,0 +1,117 @@
+package basicevent
+
+import (
+	"errors"
+	"github.com/golang/glog"
+	"time"
+
+	"github.com/mishudark/triper"
+)
+
+//ErrSubjectMissing When an event has no subject id
+var ErrSubjectMissing = errors.New("SubjectId not found")
+var ErrCreatedAtMissing = errors.New("CreatedAt not found")
```

Comment:
both declarations could be inside the same `var` block

```
var (
        ErrSubjectMissing = errors.New("SubjectId not found")
        ErrCreatedAtMissing = errors.New("CreatedAt not found")
)
```

Reply:
I agree :)